### PR TITLE
Update Error Entity data

### PR DESCRIPTION
**Pull request by BassirouRabo**
response is of type HttpEntity, it should be converted to HttpEntity.toStrict

### PR checklist
 
- [ ] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [ ] Pull Request title clearly describes the work in the pull request and Pull Request description provides details about how to validate the work. Missing information here may result in delayed response from the community.
- [ ] Run the following to [build the project](https://github.com/OpenAPITools/openapi-generator#14---build-projects) and update samples:
  ```
  ./mvnw clean package 
  ./bin/generate-samples.sh
  ./bin/utils/export_docs_generators.sh
  ``` 
  Commit all changed files. 
  This is important, as CI jobs will verify _all_ generator outputs of your HEAD commit as it would merge with master. 
  These must match the expectations made by your contribution. 
  You may regenerate an individual generator by passing the relevant config(s) as an argument to the script, for example `./bin/generate-samples.sh bin/configs/java*`. 
  For Windows users, please run the script in [Git BASH](https://gitforwindows.org/).
- [ ] File the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master` (6.1.0) (minor release - breaking changes with fallbacks), `7.0.x` (breaking changes without fallbacks)
- [ ] If your PR is targeting a particular programming language, @mention the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) members, so they are more likely to review the pull request.

### CODE REVIEW

**Comment by wing328** [comment]
@brabo-hi thanks for the PR. Can you please follow step 3 in the PR checklist to update the samples?

**Comment by brabo-hi** [comment]
> @brabo-hi thanks for the PR. Can you please follow step 3 in the PR checklist to update the samples?

Yes I will fix it and also update Akka libraries

**Comment by wing328** [comment]
cc @clasnake (2017/07), @jimschubert (2017/09) ❤️, @shijinkui (2018/01), @ramzimaalej (2018/03), @chameleon82 (2020/03), @Bouillie (2020/04)

**Comment by ramzimaalej** [comment]
@brabo-hi Can you please document the reason why this change is required?

File: samples/client/petstore/scala-akka/src/main/scala/org/openapitools/client/core/ApiInvoker.scala [hunk 1]
```diff
@@ -268,11 +268,20 @@ class ApiInvoker(formats: Formats)(implicit system: ActorSystem) extends CustomC
         Unmarshal(response.entity)
           .to[T]
           .recoverWith {
-            case e => throw ApiError(response.status.intValue, s"Unable to unmarshall content to [$manifest]", Some(response.entity.toString), e)
+            case e => 
+            response
+                .entity
+                .toStrict(FiniteDuration(6, TimeUnit.SECONDS))
```
Comment (chameleon82) [inline samples/client/petstore/scala-akka/src/main/scala/org/openapitools/client/core/ApiInvoker.scala:274]: It seems magic number here. Why 6 seconds was choosen?

**Pull request closed without merging**